library/src/main/java/com/bumptech/glide/load/resource/gif/GifDrawable.java

```diff
@@ -66,6 +70,9 @@ public class GifDrawable extends Drawable
 
   private boolean applyGravity;
   private Paint paint;
+  private ColorStateList tint;
+  private PorterDuff.Mode tintMode;
```

Comment:
Default tintMode is documented as SRC_IN

https://developer.android.com/reference/android/graphics/drawable/Drawable#setTintMode(android.graphics.PorterDuff.Mode)

The Android Drawable implementations also imply that this should not be settable to null but null should set back to the default (this behavior is not documented and is not marked as nullable, but the base Drawable compat implementation treats null as default values)

Reply:
> null should set back to the default

You are right. I updated this PR to use the default mode if null.

> The Android Drawable implementations also imply that this should not be settable to null

FYI, Drawable#setTintMode() accept null.
https://cs.android.com/android/platform/superproject/+/master:frameworks/base/graphics/java/android/graphics/drawable/Drawable.java;l=692?q=drawable.java